easybuild/easyblocks/e/esmf.py

```diff
@@ -73,7 +72,12 @@ def configure_step(self):
 
         # specify decent LAPACK lib
         env.setvar('ESMF_LAPACK', 'user')
-        env.setvar('ESMF_LAPACK_LIBS', '%s %s' % (os.getenv('LDFLAGS'), os.getenv('LIBLAPACK_MT')))
+        ldflags = os.getenv('LDFLAGS')
+        liblapack_mt = os.getenv('LIBLAPACK_MT')
+        if liblapack_mt is None:
+            raise EasyBuildError("$LIBLAPACK_MT not defined, no BLAS/LAPACK in %s toolchain?", self.toolchain.name)
```

Comment:
This is technically wrong.
If LIBLAPACK_MT is not defined it should just use LIBLAPACK
I.e., if the easyconfig didn't set openmp: True use LIBLAPACK, if it did use openmp: True use LIB LAPACK_MT

Reply:
The ESMF easyconfig doesn't set `openmp` at all, see https://github.com/easybuilders/easybuild-easyconfigs/pull/9513/files .

Related discussion: https://github.com/easybuilders/easybuild-framework/issues/2350

Reply:
Ah, I'd forgotten about that thread.

Reply:
@akesandgren How about if I use this instead:

```python
liblapack = os.getenv('LIBLAPACK_MT') or os.getenv('LIBLAPACK')

Reply:
Yeah, that makes sense. Although I still think we should get something in the way of 2350 in place.